processor/groupbyattrsprocessor/README.md

```diff
@@ -83,6 +86,64 @@ Notes:
 * The specified "grouping" attributes that are set on the new *Resources* are also **removed** from the metric *DataPoints*
 * While not shown in the above example, the processor also merges collections of records under matching InstrumentationLibrary
 
+### Compaction
+
+In some cases, the data might come in single requests to the collector and even after batching there might be multiple duplicated ResourceSpans/ResourceLogs/ResourceMetrics objects, which leads to additional memory consumption and increased processing costs. As a remedy, `groupbyattrs` processor might be used to compact the data which has matching Resource and InstrumentationLibrary properties.
```

Comment:
To me, the appalling aspect of this feature is to get better performance while sending data out. Without calling this out explicitly, people might not realize that the advantages are good on the transport side as well.

Reply:
If I understand correctly, this will reduce the size of the message for many of the formats (e.g. OTLP). In some cases (Jaeger) this will also reduce the number of RPC calls (since the Jaeger model maps one batch to one ResourceSpans) - if I got it right. Perhaps it would be worth calling out in jaegerexporter that `groupbyattrs` is recommended (or worth considering)?